Asset name of darwin/amd64 is missing the version

https://github.com/mattn/bsky/releases/tag/v0.0.63

[bsky-darwin-.zip](https://github.com/mattn/bsky/releases/download/v0.0.63/bsky-darwin-.zip)

<img width="638" alt="image" src="https://github.com/user-attachments/assets/36aaf4c6-e84b-4d62-9b6a-5f9367b34d32">

## Note

- https://github.com/aquaproj/aqua-registry/pull/29219

CLI Version Manager [aqua](https://aquaproj.github.io) supports bsky, but it failed to install bsky v0.0.63 on macOS, so I noticed this issue.

https://github.com/mattn/bsky/actions/runs/12020901190/job/33510365063#step:5:140

```
make[1]: gobump: No such file or directory
make[1]: *** [show-version] Error 1
```

make show-version failed because `gobump` isn't found.

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -38,7 +38,7 @@ jobs:
 
   build_for_macos:
     name: Build for MacOS
-    runs-on: macos-latest
+    runs-on: macos-13 # Use macos-13 to build darwin/amd64. macos-latest is arm64.
     steps:
     - name: Install build dependencies
       run: brew install coreutils

